src/svgutils/transform.py

```diff
@@ -49,6 +49,45 @@ def rotate(self, angle, x=0, y=0):
         self.root.set("transform", "%s rotate(%f %f %f)" %
                       (self.root.get("transform") or '', angle, x, y))
 
+    def skew_x(self, angle):
```

Comment:
Sure– I can move `skew` out of the `Element` class and into `FigureElement`, and eliminate the boilerplate.